src/sources/journald.rs

```diff
@@ -97,6 +97,11 @@ impl SourceConfig for JournaldConfig {
         shutdown: ShutdownSignal,
         out: Pipeline,
     ) -> crate::Result<super::Source> {
+        if self.remap_priority {
+            error!("Option `remap_priority` has been deprecated. You can use `remap` transform and function `to_syslog_level` instead.");
```

Comment:
I think we may want to make this a warning first, then an error in a later version. Unless `remap_priority` was never released.

Reply:
Note that this change actually removes the functionality as well. So, the question is if this should be left as an error, or changed to a warning *and the functionality restored*.

Reply:
:+1: I think it would be prudent to deprecate first and then remove in a later release.

Reply:
Sounds like a good plan.